runtime/litentry/src/weights/pallet_proxy.rs

```diff
@@ -138,20 +138,18 @@ impl<T: frame_system::Config> pallet_proxy::WeightInfo for WeightInfo<T> {
 	}
 	// Storage: unknown [0x3a65787472696e7369635f696e646578] (r:1 w:0)
 	// Storage: Proxy Proxies (r:1 w:1)
-	/// The range of component `p` is `[1, 31]`.
-	fn anonymous(p: u32, ) -> Weight {
-		Weight::from_ref_time(35_206_000 as u64)
-			// Standard Error: 1_000
-			.saturating_add(Weight::from_ref_time(61_000 as u64).saturating_mul(p as u64))
+	fn create_pure(p: u32, ) -> Weight {
+		Weight::from_ref_time(31_077_000 as u64)
+			// Standard Error: 3_000
+			.saturating_add(Weight::from_ref_time(37_000 as u64).saturating_mul(p as u64))
```

Comment:
I see in the runtime setting `()` is used so it probably won't have an impact. But still, this doesn't look like automatically generated